client/my-sites/store/style.scss

```diff
@@ -2,18 +2,15 @@
 @import "calypso/my-sites/stats/modernized-layout-styles";
 
 .woocommerce {
-	@import "app/dashboard/style";
 	@import "app/store-stats/store-stats-chart/style";
 	@import "app/store-stats/store-stats-module/style";
 	@import "app/store-stats/store-stats-widget-list/style";
 	@import "app/store-stats/style";
 
-	@import "components/action-header/style";
 	@import "components/d3/base/style";
 	@import "components/d3/sparkline/style";
 	@import "components/delta/style";
 	@import "components/table/style";
-	@import "components/woocommerce-colophon/style";
 
 	.main {
 		padding-top: 72px;
```

Comment:
The `woocommerce__placeholder` styles could also be removed, I don't see any usage.